arthur_bench/client/local/client.py

```diff
@@ -396,22 +396,22 @@ def get_runs_for_test_suite(
     def get_summary_statistics(
         self,
         test_suite_id: str,
-        run_id: Optional[str] = None,
+        run_ids: Optional[list[str]] = None,
         page: int = 1,
         page_size: int = DEFAULT_PAGE_SIZE,
     ) -> TestSuiteSummary:
         test_suite_name = self._get_suite_name_from_id(test_suite_id)
         if test_suite_name is None:
             raise NotFoundError(f"no test suite with id {test_suite_id}")
 
-        runs = []
-        run_id_found = False
+        runs: list[SummaryItem] = []
         run_files = glob.glob(f"{self.root_dir}/{test_suite_name}/*/run.json")
         for f in run_files:
             run_obj = PaginatedRun.parse_file(f)
             runs.append(_summarize_run(run=run_obj))
-            if str(run_obj.id) == run_id:
-                run_id_found = True
+
+        if run_ids:
```

Comment:
Could be worth moving this check above to avoid reading runs necessarily

Reply:
Hm this seems like it is currently broken if there are no runs, surprised mypy didn't catch that

Reply:
You could get the number of test cases from the test suite like so:
```
suite_file = self.root_dir / test_suite_name / "suite.json"
suite = PaginatedTestSuite.parse_file(suite_file)
num_test_cases = len(suite.test_cases)
```

This will read the full file into a single page

Reply:
Let me know if you want to fix here, or I can file a backlog bug ticket for this and pick up separately

Reply:
Sorry, forgot about this thread. Updated here